src/components/POLine/Eresources/EresourcesView.js

```diff
@@ -42,7 +42,7 @@ export default function EresourcesView({ line: { eresource }, order, vendors })
       </Col>
       <Col xs={3}>
         <KeyValue label={<FormattedMessage id="ui-orders.eresource.createItem" />}>
-          <Checkbox checked={get(eresource, 'create_inventory')} disabled />
+          <Checkbox checked={get(eresource, 'createInventory')} disabled />
```

Comment:
Did you miss user_limit below (line 61)?

Reply:
missed, thanks!